www/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: [
+    '@pluralsh/eslint-config-typescript',
+  ],
+  globals: {
+    JSX: true,
```

Comment:
I guess we can add that directly to the global config instead.

Reply:
Sure!